reference/tests/snapshot.rs

```diff
@@ -93,11 +96,14 @@ fn write_reference_examples() -> Result<()> {
 fn run_reference_examples() -> Result<()> {
     glob!("examples/**/*.prql", |path| {
         let prql = fs::read_to_string(path).unwrap();
-
-        if prql.contains("skip_test") {
-            return;
-        }
-
+        dbg!(&prql);
+        let sql = compile(&prql).unwrap_or_else(|e| format!("Failed to compile `{prql}`; {e}"));
+        assert_snapshot!(sql);
+    });
+    write_reference_examples();
+    glob!("examples/**/*.prql", |path| {
+        let prql = fs::read_to_string(path).unwrap();
+        dbg!(&prql);
```

Comment:
The proximate cause is it failing after _writing_ the files. So normal snapshots work. But because we generate the `.prql` files from the docs as part of the test, it's somehow getting confused with line endings.

Possibly it's writing them with `CRLF`, rather than `LF`, and insta is failing to read those correctly? I'm still not sure at this level